app/kumactl/pkg/client/api_server_client.go

```diff
@@ -1,18 +1,28 @@
 package client
 
 import (
+	"fmt"
 	"net/http"
 	"net/url"
+	"runtime"
 	"time"
 
 	"github.com/pkg/errors"
 
 	config_proto "github.com/kumahq/kuma/pkg/config/app/kumactl/v1alpha1"
 	util_http "github.com/kumahq/kuma/pkg/util/http"
+	kuma_version "github.com/kumahq/kuma/pkg/version"
 )
 
 func ApiServerClient(coordinates *config_proto.ControlPlaneCoordinates_ApiServer, timeout time.Duration) (util_http.Client, error) {
-	headers := make(map[string]string)
+	headers := map[string]string{
+		"User-Agent": fmt.Sprintf("kumactl/%s (%s; %s; %s/%s)",
+			kuma_version.Build.Version,
+			runtime.GOOS,
+			runtime.GOARCH,
+			kuma_version.Build.Product,
+			kuma_version.Build.GitCommit[:7]),
```

Comment:
Do you want the full commit or the short version is good enough ?

Reply:
short is good enough